lakey_service/catalogue/templatetags/catalogue_tags.py

```diff
@@ -0,0 +1,53 @@
+
+
+import pandas as pd
+
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+from django import template
+from django.utils.safestring import mark_safe
+
+
+register = template.Library()
+
+
+@register.simple_tag
+def sample_table(sample):
+    print(f"sample: {sample}")
```

Comment:
print is not needed ;-)

Reply:
:roll_eyes: missed this one :stuck_out_tongue: